dll/shellext/zipfldr/zippidl.hpp

```diff
@@ -15,20 +16,20 @@ enum ZipPidlType
 #include <pshpack1.h>
 struct ZipPidlEntry
 {
-    WORD cb;
-    BYTE MagicType;
+    USHORT cb; // This must be a USHORT for SHITEMID compatibility
+    WORD MagicType;
     ZipPidlType ZipType;
 
     ULONG64 CompressedSize;
     ULONG64 UncompressedSize;
     ULONG DosDate;
     BYTE Password;
+    BYTE Utf8;
```

Comment:
Why is this info needed?
It is stored as WCHAR anyway.

Reply:
Deleted. Some members are reordered.